mgmtsystem_audit/i18n/fr.po

```diff
@@ -429,36 +349,88 @@ msgid "To Improve Points"
 msgstr "Points à améliorer"
 
 #. module: mgmtsystem_audit
-#: field:mgmtsystem.audit,message_unread:0
-msgid "Unread Messages"
-msgstr "Messages non-lus"
+#: help:mgmtsystem.audit,message_summary:0
+msgid ""
+"Holds the Chatter summary (number of messages, ...). This summary is "
+"directly in html format in order to be inserted in kanban views."
+msgstr ""
+"Contient le résumé de la discussion (nombre de messages, ...). Ce résumé "
+"est au format HTML pour permettre son utilisation dans les vues kanban."
+
+#. module: mgmtsystem_audit
+#: model:ir.actions.server,name:mgmtsystem_audit.action_email_send_verification_list
+msgid "Send Verification List"
+msgstr "Envoyer la liste de vérification"
+
+#. module: mgmtsystem_audit
+#: view:copy.verification.lines:0
+msgid "_Cancel"
+msgstr "Annuler"
 
 #. module: mgmtsystem_audit
-#: code:_description:0
 #: model:ir.model,name:mgmtsystem_audit.model_mgmtsystem_verification_line
 #: view:mgmtsystem.verification.line:0
-#, python-format
 msgid "Verification Line"
 msgstr "Ligne de vérification"
 
 #. module: mgmtsystem_audit
-#: model:ir.actions.act_window,name:mgmtsystem_audit.open_mgmtsystem_verification_line_list
-#: view:mgmtsystem.verification.line:0
-msgid "Verification Lines"
-msgstr "Lignes de vérification"
+#: view:mgmtsystem.audit:0
+msgid "Close"
+msgstr "Fermer"
 
 #. module: mgmtsystem_audit
-#: view:mgmtsystem.audit:0 field:mgmtsystem.audit,line_ids:0
-#: report:mgmtsystem.audit.verificationlist:0
-msgid "Verification List"
-msgstr "Liste de vérification"
+#: model:ir.actions.act_window,name:mgmtsystem_audit.open_mgmtsystem_audit_list
+#: model:ir.ui.menu,name:mgmtsystem_audit.menu_open_audit
+#: view:mgmtsystem.audit:0
+msgid "Audits"
+msgstr "Audits"
 
 #. module: mgmtsystem_audit
-#: view:copy.verification.lines:0
-msgid "_Cancel"
-msgstr "Annuler"
+#: report:mgmtsystem.audit.report:0
+#: report:mgmtsystem.audit.verificationlist:0
+msgid "Auditors:"
+msgstr "Auditeurs:"
 
 #. module: mgmtsystem_audit
-#: view:copy.verification.lines:0
-msgid "_Copy"
-msgstr "_Copier"
+#: view:mgmtsystem.audit:0 field:mgmtsystem.audit,auditee_user_ids:0
+msgid "Auditees"
+msgstr "Audités"
+
+#~ msgid "Action Plan Comments"
```

Comment:
Ah I'd guess the automatic suggestions would be stored in a different place. If it can help translate back already translated strings, then it's fine.